solution_python.py

```diff
@@ -1,23 +1,80 @@
+from enum import Enum
+
+class Events(Enum):
+    ADD = 1
+    SUBTRACT = 2
+    MULTIPLY = 3
+    DIVIDE = 4
+    POW = 5
+
+class Event():
+    def __init__(self, value, change, event):
+        self.value = value
+        self.change = change
+        self.event = event
+
 class EventSourcer():
     # Do not change the signature of any functions
 
+    EVENT_FUNCTIONS = {
```

Comment:
Reflecting on this, I'm not sure if I should have put it in `Events` and make it a static class rather than an enum. That way all arithmetic would be in one place.